numpy/core/src/umath/ufunc_object.c

```diff
@@ -1961,8 +2122,29 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
         goto fail;
     }
 
+    if (axes) {
+        /*
+         * possibly remap axes, using newly allocated memory.
+         */
+        remap_axis = (int **)PyArray_malloc(sizeof(int *) * nop);
```

Comment:
Better to write `sizeof(remap_axis[0]) * nop`, and below `sizeof(remap_axis_memory[0]) * ...`. stackoverflow suggests dropping the cast too.

Reply:
I vaguely recall having problems without the cast, but cannot reproduce those, so it must have been something else. Now changed per your suggestion (much better).